packages/bap-goodies/bap-goodies.0.1/files/Makefile

```diff
@@ -0,0 +1,30 @@
+.PHONY: mkpath install uninstall
+
+OPAMSHARE = $(shell opam config var share)/emacs/site-lisp
+
+bap-mode.el:
+	@test -f bap-mode.el || \
+	wget  https://raw.githubusercontent.com/fkie-cad/bap-mode/master/bap-mode.el
```

Comment:
this will not work in OPAM 2.0, it won't allow you to perform network operations during the installation phase. Please, specify the package properly.